### ISSUE TITLE
Add videos to recent files

### ISSUE THREAD

**Issue by Exalm** [SeaDve/Kooha#126]
**System Info**
- 2.0.0.0, Flatpak

**Describe the bug**
Videos aren't added to recent files. See https://gitlab.gnome.org/GNOME/gnome-shell/-/merge_requests/1439/diffs for a similar thing in shell previously. It also worked in  the previous version.

I thought it might be that it's written inside the sandbox instead, but that's not it. - the recent files xml within sandbox doesn't even exist.

**Comment by SeaDve** [comment]
Implemented it with 75835f4960b913006b9e156695114bc675eef35c, but I think it is not getting through the sandbox. It works before because gnome-shell handled it

**Comment by Exalm** [comment]
Yeah, possible. The weird thing though is that there's no `recently-used.xbel` in the sandbox either.

**Comment by SeaDve** [comment]
Ah, that was a new commit (a few minutes from the reply), so maybe that. I checked after that commit, it gets added now on `recently-used.xbel` of the sandbox
